src/remove.js

```diff
@@ -0,0 +1,112 @@
+import { taskCompleted } from './checkbox.js';
+
+const addTask = (e, items, input, itemsContainer, Item) => {
+  items = [...JSON.parse(localStorage.getItem('items'))];
+  if (e.keyCode === 13) {
+    const newItem = new Item();
+    const div = document.createElement('div');
+    const checkbox = document.createElement('input');
+    const p = document.createElement('input');
+    const icon = document.createElement('i');
+    div.classList.add('task');
+
+    newItem.description = input.value;
+    newItem.id = items.length + 1;
+    newItem.completed = false;
+
+    checkbox.type = 'checkbox';
+    checkbox.classList.add('checkbox');
+    checkbox.addEventListener('change', taskCompleted);
+
+    p.value = input.value;
+    p.type = 'text';
+    p.setAttribute('readonly', 'readonly');
+
+    icon.classList.add('fas', 'fa-ellipsis-v', 'flex-end');
+
+    div.appendChild(checkbox);
+    div.appendChild(p);
+    div.appendChild(icon);
+
+    itemsContainer.appendChild(div);
+
+    input.value = '';
+    items.push(newItem);
+    localStorage.setItem('items', JSON.stringify(items));
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
+  }
+};
+
+const removeTasks = (e, items) => {
+  items = [...JSON.parse(localStorage.getItem('items'))];
+  const checked = document.querySelectorAll('input[type="checkbox"]');
+  checked.forEach((checkbox) => {
+    if (checkbox.checked) {
+      checkbox.parentElement.remove();
+    }
+  });
+
+  for (let i = 0; i < items.length; i += 1) {
+    items.filter((item) => {
+      if (item.completed) {
+        const index = items.indexOf(item);
+        items.splice(index, 1);
+        let i = 0;
+        while (i < items.length) {
+          if (items[i].id > item.id) {
+            items[i].id -= 1;
+          }
+          i += 1;
+        }
+        localStorage.setItem('items', JSON.stringify(items));
```

Comment:
- [ ] The delete option for the `checkbox` is working but it should also take the `clear all completed`.
- [ ] Also the `index` of each added array does not change after removing one from the array.

![Capture web_1-10-2021_16404_gitlab com](https://user-images.githubusercontent.com/61562626/135649171-297f2ede-f4ae-4a09-9580-966bdae41c90.jpeg)

![mmmm](https://user-images.githubusercontent.com/61562626/135649736-48fd9a96-3132-4937-bc0c-d4427cee7e1c.jpeg)